client/tsconfig.json

```diff
@@ -16,5 +16,12 @@
 		}
 	},
 	"references": [ { "path": "../packages" }, { "path": "../build-tools" } ],
+	"include": [
+		"**/*",
+		"./lib/domains/tlds/wpcom-multi-level-tlds.json",
```

Comment:
JSON files have to be included explicitly. I couldn't include `./cached-requests.json` because it is outside `rootDir`